### PR TITLE
Update error-codes.mdx

### PR DESCRIPTION
**Pull request by Ptitet**
Removed mention to old setting `max` to use `limit`

### CODE REVIEW

File: docs/reference/error-codes.mdx [hunk 1]
```diff
@@ -286,7 +286,7 @@ Set `validate: {creationStack: false}` in the options to disable the check.
 > Added in `6.10.0`.
 
 In express-rate-limit version 6 and older, the rate limiter would be disabled
-when setting `limit` (now `max`) to `0` in the options. Starting with version
+when setting `limit` to `0` in the options. Starting with version
```
Comment (nfriedly) [inline docs/reference/error-codes.mdx:289]: Oh 🤦 

Those two were supposed to be flipped. It's talking about older versions in that sentence, so it should read "max (now limit)"

```suggestion
when setting `max` (now `limit`) to `0` in the options. Starting with version
```

**Comment by Ptitet** [comment]
I thought about this but I wasn't sure.
Is it really necessary to keep the `max` ? It's deprecated so people should use `limit`, and the help to know that they are the same should be in the doc for `limit` no ?

**Comment by nfriedly** [comment]
Yeah, looking at the [npm download stats](https://www.npmjs.com/package/express-rate-limit?activeTab=versions), the majority of users are still on older versions, so I think we should keep the mention of `max` in the docs for now.

Also the error code has `MAX` in it, because it predates the rename.

**Comment by Ptitet** [comment]
@nfriedly what about "setting `limit` (previously `max`)" ?

**Comment by nfriedly** [comment]
> what about "setting limit (previously max)" ?

Yeah, I like that 

> We can even specify that this was until release 7.0.0.

Also a good idea.

**Comment by Ptitet** [comment]
I’ve made a new commit to reflect what we discussed. Does it look like what you expected?

**Review by nfriedly — APPROVED** [review]
Yep, this looks good, thanks!

**Comment by nfriedly** [comment]
Merged in [ff3702e](https://github.com/express-rate-limit/express-rate-limit/commit/ff3702e4d92c2ab93022d8f66e0c3786c3b8d799)

(Sorry, I did it locally to fix the lint error, but github doesn't connect squash commits with the original PR, so it's going to show up as closed rather than merged.)

**Pull request closed without merging**